submissions/SofiiaTrokhymchuk/frogger-game/app.js

```diff
@@ -0,0 +1,147 @@
+function createWinLossPointsBlock(){
+    const pointsBlock = document.createElement('div');
+    pointsBlock.style.cssText = `
+        font-family: Helvetica; 
+        font-size: 30px; 
+        width: 50%;
+        margin: 30px auto -10px;
+        display: flex;
+        justify-content: space-evenly;
+    `
+    pointsBlock.innerHTML = `
+        <span>Wins: 0</span>
+        <span>Losses: 0</span>
+    `;
+    document.body.prepend(pointsBlock);
+    return pointsBlock;
+}
+
+function addWinLossPointsToBlock(pointsBlock, player){
+    pointsBlock.innerHTML = `
+        <span>Wins: ${player.wins}</span>
+        <span>Losses: ${player.loses}</span>
+    `;
+}
+
+const pointsBlock = createWinLossPointsBlock();
+
+const TILE_WIDTH = 101;
+const TILE_HEIGHT = 83;
+
+const BOARD_WIDTH = TILE_WIDTH * 4;
+const BOARD_HEIGHT = TILE_HEIGHT * 5;
+
+const CHARACTERS_WIDTH = 101;
+const CHARACHTERS_HEIGHT = 171;
+
+const SIZE_DIFFERENCE = 20;
```

Comment:
Size of what this difference is?

Reply:
If I place a player and a bug at the same postion, they still look like one of them is higher. So, I use this const to align characters a little bit. Maybe I should rename it to something like `CHARACTERS_SIZE_DIFFERENCE`.

Reply:
`CHARACTERS_SIZE_DIFFERENCE` explains things better. Is it any Does it refer both dimensions or only height/width?

Reply:
Then that's fine. At times it makes sense to add redundancy for the sake of clarity.
You may skip fixing this.

Reply:
I've already changed this const name, should I rename it back, or it's fine to leave it as `CHARACTERS_SIZE_DIFFERENCE`?

Reply:
If the naming became more descriptive then keep it.